lib/licenses.nix

```diff
@@ -266,6 +266,12 @@ lib.mapAttrs (n: v: v // { shortName = n; }) {
     free = false;
   };
 
+  erfa = {
+    fullName = "ERFA License";
+    url = https://github.com/liberfa/erfa/blob/master/LICENSE;
```

Comment:
```suggestion
    url = "https://github.com/liberfa/erfa/blob/master/LICENSE";
```

Reply:
Context: https://github.com/NixOS/rfcs/pull/45

We should have a linter for this.